content/en/community/hackathons/sessions/bincompat/content/Work-Items_00.-Setup.md

```diff
@@ -1,25 +1,89 @@
-To easily setup, build and run Linux ELFs with [`app-elfloader`](https://github.com/unikraft/app-elfloader), best way is to use [the `scripts` repository](https://github.com/unikraft-upb/scripts).
-Clone [the `scripts` repository](https://github.com/unikraft-upb/scripts) on your machine to get started.
+To easily setup, build and run Linux ELFs with [`app-elfloader`](https://github.com/unikraft/app-elfloader), best way is to use [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
+Clone [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader) on your machine to get started.
+Along with the `run-app-elfloader` repository, we have collected some already compiler applications that you can use to test the `elfloader`.
+Those are located in the [`static-pie-apps`](https://github.com/unikraft/static-pie-apps/) and [`dynamic-apps`](https://github.com/unikraft/dynamic-apps/) repositories.
+It's just a bunch of already compiled applications, so you don't have to spend time compiling stuff, there is nothing Unikraft-related in those repositories.
 
 ```console
-$ git clone https://github.com/unikraft-upb/scripts
+$ git clone https://github.com/unikraft/run-app-elfloader
+$ git clone https://github.com/unikraft/static-pie-apps
+$ git clone https://github.com/unikraft/dynamic-apps
 
-$ cd scripts/
+$ cd run-app-elfloader/
 
-$ cd make-based/app-elfloader/
+$ ./run_app.sh helloworld
 
-$ ./do.sh setup
+SeaBIOS (version rel-1.16.2-0-gea1b7a073390-prebuilt.qemu.org)
+Booting from ROM..TEST nofollow
+Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                  Atlas 0.13.1~d20aa7cb
+[...]
+brk(NULL) = va:0x400200000
+brk(va:0x400221000) = va:0x400221000
+Hello, World!
+write(fd:1, "Hello, World!\x0A", 14) = 14
+qemu-system-x86_64: terminating on signal 2
+```
+
+This will run a dynamically linked `helloworld` application.
+You will see a lot of output because, by default, the `run_app.sh` script uses a prebuilt version of `app-elfloader`, which has `strace`-like output enabled, for debugging purposes.
+You can see a list of all existing available application that you can run in binary compatibility mode by running `./run_app.sh` with no arguments.
+**Note: If you are running an application that requires networking support, you will need to run it with `sudo`.***
+
+If you want more control over the application, you can use the `run.sh` script.
+The `elfloader` application will need to be passed a filesystem that contains all the dependencies (i.e. external libraries) the already compiled application needs, along with the [system loader](https://www.man7.org/linux/man-pages/man8/ld.so.8.html), and will expect as an argument the name of the `ELF` that will be executed.
```

Comment:
This is a convoluted phrase. Break it in smaller sentences, reword where necessary (i.e. `need` / `needs` appears twice, `will` appears multiple times).